hayeon/baekjoon/브루트포스/7568-덩치.js

```diff
@@ -0,0 +1,42 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  const test = input.shift();
+  const sizeTable = input.map((size) =>
+    size.split(" ").map((num) => parseInt(num))
+  );
+  const rank = [];
+
+  for (let i = 0; i < test; i++) {
+    let count = 0;
+
+    for (let j = 0; j < test; j++) {
+      if (i !== j) {
+        if (
+          sizeTable[i][0] < sizeTable[j][0] &&
+          sizeTable[i][1] < sizeTable[j][1]
+        ) {
+          count++;
+        }
+      }
+    }
+    rank.push(count + 1);
+  }
+  console.log(rank.join(" "));
```

Comment:
```suggestion
  const test = +input.shift();
  const sizeTable = input.map((size) =>size.split(" ").map(Number));
  const rank = [];

  for (let i = 0; i < test; i++) {
    let count = 0;

    for (let j = 0; j < test; j++) {
      if (i === j) continue;
      if (
        sizeTable[i][0] < sizeTable[j][0] &&
        sizeTable[i][1] < sizeTable[j][1]
      ) {
        count++;
      }
    }
    rank.push(count + 1);
  }
  console.log(rank.join(" "));
```
- 잘봣습니다~ 🙇‍♂️
- 기존에 입력값을 형변환할 때 map(Number)로 하셨는데 기존처럼 하는게 좀 더 깔끔해보입니다. ~
- 그리고 i === j 일 때 continue를 해줌으로써 중첩 조건문을 피하는 방법이 더 깔끔한 것 같습니다 :)

Reply:
오옷 넵넵 피드백 감사합니다!! 고민하던 부분이었는데 훨씬 더 깔끔해보이네요!!!! 👍🏻